pkg/kdo/cli/component/build.go

```
@@ -78,113 +73,228 @@ func (o *BuildIDPOptions) Run() (err error) {
-	if err != nil {
-		err = errors.New("The build job failed to run")
-		return
-	}
+		// Execute the Mvm command in the Build Container
+		command := []string{"/bin/sh", "-c", string(build.FullBuildTask)}
+		if o.buildTaskType == string(build.Incremental) {
+			command = []string{"/bin/sh", "-c", string(build.IncrementalBuildTask)}
+		}
+		// command := []string{"/bin/sh", "-c", "hostname", "-f"}
+		output, stderr, err := o.Context.Client.ExecPodCmd(command, ReusableBuildContainerInstance.ContainerName, ReusableBuildContainerInstance.PodName)
+		if len(stderr) != 0 {
+			fmt.Println("Reusable Build Container stderr: ", stderr)
+		}
+		if err != nil {
+			fmt.Printf("Error occured while executing command %s in the pod %s: %s\n", strings.Join(command, " "), ReusableBuildContainerInstance.PodName, err)
+			os.Exit(1)
+		} else {
+			fmt.Printf("Reusable Build Container Output: \n%s\n", output)
+		}
 
-	err = o.Context.Client.GetPodLogs(po, os.Stdout)
-	if err != nil {
-		err = errors.New("Failed to get the logs for the build job")
-		return
-	}
+		fmt.Println("Finished executing the IDP Build Task in the Reusable Build Container...")
+	} else {
+		// Create a Kube Job for building
+		fmt.Println("Creating a Kube Job for building...")
+
+		buildTaskJobName := "codewind-liberty-build-job"
+
+		job, err := build.CreateBuildTaskKubeJob(buildTaskJobName, o.buildTaskType, namespace, idpClaimName, "projects/"+o.projectName, o.projectName)
+		if err != nil {
+			fmt.Println("There was a problem with the job configuration, exiting...")
+			panic(err.Error())
+		}
 
-	// TODO-KDO: Set owner references
-	var jobSucceeded bool
-	// Loop and see if the job either succeeded or failed
-	loop := true
-	for loop == true {
-		jobs, err := clientset.BatchV1().Jobs(namespace).List(metav1.ListOptions{})
+		kubeJob, err := clientset.BatchV1().Jobs(namespace).Create(job)
 		if err != nil {
+			fmt.Println("Failed to create a job, exiting...")
 			panic(err.Error())
```

Can we return the error instead of the panic?